lib/rpm_package.py

```diff
@@ -205,6 +205,11 @@ def _load(self):
                     dep_name, self.distro, force_rebuild=self.force_rebuild)
                 self.build_dependencies.append(dep)
 
+            # keeps backward compatibility with old yaml files which have build dependencies
```

Comment:
I understand that some code slipped into the tree without complying with PEP 8.  Perhaps that's a sign that we need to improve our automated checks and code validations.

If we have the opportunity to comply with PEP 8 during review, it's better for us.  This will result in less work in the future and code will get fixed gradually.